robots/LoCoBot/locobot_control/nodes/pointrobot3factor_ros_server.py

```diff
@@ -121,16 +121,12 @@ def _odometry_callback(self, msg):
         self.state.pose = np.asarray([x, y, yaw])
         self.state.vel = np.asarray([x_vel, y_vel, ang_vel])
 
-    def set_vel(self, vel, exe_time):
```

Comment:
The only reason i did this was because my robot object in the the `pointrobot3factor_ros_server.py` is a light-weight object. I did not want to invoke the whoel `pyrobot.Robot` object there.